packages/types/src/order/service.ts

```diff
@@ -1371,4 +1386,24 @@ export interface IOrderModuleService extends IModuleService {
     config?: RestoreReturn<TReturnableLinkableKeys>,
     sharedContext?: Context
   ): Promise<Record<TReturnableLinkableKeys, string[]> | void>
+
+  revertLastVersion(orderId: string, sharedContext?: Context): Promise<void>
+
+  restoreNextVersion(orderId: string, sharedContext?: Context): Promise<void>
```

Comment:
question: Do we need these to be part of the public interface?

I would have imagined that the methods we expose would be more action-specific and use these under the hood.

Reply:
I pushed a version fixing this. To me the revertLastVersion can be public but it doesn't make sense to restore that.
this will act as a soft delete of the latest changes applied

Reply:
it wasn't intentional, but I guess I fixed a couple of things related to this PR as well. Like adding `deleted_at` in places where they weren't present.
This module will need a cleanup after most of the functionalities are in place.

Reply:
Cool, I am fine to keep those changes in